src/pkg/db/dependency_attributes.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) CloudCover
+// SPDX-License-Identifier: Apache-2.0
+
+package db
+
+import (
+	"github.com/cldcvr/terrarium/src/pkg/jsonschema"
+	"github.com/cldcvr/terrarium/src/pkg/pb/terrariumpb"
+	"github.com/google/uuid"
+)
+
+type DependencyAttribute struct {
+	Model
+
+	DependencyID *uuid.UUID       `gorm:"uniqueIndex:dependency_attribute_unique"`
+	Name         *string          `gorm:"uniqueIndex:dependency_attribute_unique"`
+	Schema       *jsonschema.Node `gorm:"type:jsonb"`
+	Computed     *bool            `gorm:"uniqueIndex:dependency_attribute_unique"` // true means output, false means input
```

Comment:
Is there a reason for making each field pointer here? This is not consistent with the other Schemas including "Dependency".

```suggestion
	DependencyID uuid.UUID       `gorm:"uniqueIndex:dependency_attribute_unique"`
	Name         string          `gorm:"uniqueIndex:dependency_attribute_unique"`
	Schema       *jsonschema.Node `gorm:"type:jsonb"`
	Computed     bool            `gorm:"uniqueIndex:dependency_attribute_unique"` // true means output, false means input
```